core/dbt/node_runners.py

```diff
@@ -70,6 +70,18 @@ def __init__(self, config, adapter, node, node_index, num_nodes):
         self.skip = False
         self.skip_cause = None
 
+    def get_result_status(self, result) -> Dict[str, str]:
+        if result.error:
+            return {'model_status': 'error', 'model_error': str(result.error)}
+        elif result.skip:
+            return {'model_status': 'skipped'}
+        elif result.fail:
+            return {'model_status': 'failed'}
+        elif result.warn:
+            return {'model_status': 'warn'}
+        else:
+            return {'model_status': 'passed'}
```

Comment:
can we change all of these from `model_status` to `node_status` for consistency? Many of these results will be generated from nodes which are not models.